projects/js-packages/webpack-config/README.md

```diff
@@ -101,8 +101,8 @@ In production mode we choose no devtool, mainly because we don't currently distr
 
 This is an object suited for spreading some default values into Webpack's `output` configuration object. We currently set two of the settings:
 
-- `filename`: `[name].min.js`. The `.min.js` bit is required to avoid a broken auto-minifier on WordPress.com infrastructure.
-- `chunkFilename`: `[name]-[id].H[contenthash:20].min.js`. The content hash serves as a cache buster; while Webpack would accept something like `[name]-[id].min.js?ver=[contenthash]`, [some of the modules we use do not](https://github.com/Automattic/jetpack/issues/21349#issuecomment-940191828).
```

Comment:
Hi @anomiex so '[some of the modules we use do not](https://github.com/Automattic/jetpack/issues/21349#issuecomment-940191828)' is no longer a problem? And also we could remove the readable js plugin now, right?

Reply:
> Hi @anomiex so 'some of the modules we use do not' is no longer a problem?

The only one I was aware of was `@automattic/webpack-rtl-plugin`, which was fixed by https://github.com/Automattic/wp-calypso/pull/57479.

> And also we could remove the readable js plugin after this, right?

This PR is already doing that. Mostly since the forked version at least was broken in a similar way to `@automattic/webpack-rtl-plugin`.